dbos/_recovery.py

```diff
@@ -4,22 +4,52 @@
 import traceback
 from typing import TYPE_CHECKING, Any, List
 
+from sqlalchemy.exc import DatabaseError
+
 from ._core import execute_workflow_by_id
-from ._error import DBOSWorkflowFunctionNotFoundError
+from ._error import DBOSWorkflowFunctionNotFoundError, DBOSWorkflowRecoveryError
+from ._sys_db import GetPendingWorkflowsOutput
 
 if TYPE_CHECKING:
     from ._dbos import DBOS, WorkflowHandle
 
 
-def startup_recovery_thread(dbos: "DBOS", workflow_ids: List[str]) -> None:
+def clear_pending_workflow_queue_assignement(
```

Comment:
No need for this wrapper function. There actually should never be a serialization error here, and if there is that means some of our assumptions are wrong, so we shouldn't be hiding those errors.